packages/pipeline/src/scripts/create-tables.ts

```diff
@@ -0,0 +1,75 @@
+import { redshiftClient } from '../redshift';
+
+export const createTableScripts = {
+    createEventsTable(): any {
```

Comment:
any

Reply:
@tomhschmidt we try and avoid using `any` as much as possible since it can introduce bugs that TS cannot help us with. Please declare a type for this return value.